pytket/pytket/_tket/circuit.pyi

```diff
@@ -471,20 +471,6 @@ class Circuit:
         :return: the new :py:class:`Circuit`
         """
     @typing.overload
```

Comment:
The stub generation removes this, will look into this again tomorrow.

Reply:
yes I think I mistakenly put those in manually, should be taken out and generated I think?

Reply:
Should be working now

Reply:
thank you!